### ISSUE TITLE
muongun - UNIX api violation (malloc) (Trac #1374)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1366]
http://software.icecube.wisc.edu/static_analysis/00_LATEST/report-c5454a.html#EndPath

The behavior of `malloc()` when allocating zero bytes is implementation defined. This could result in this section of code **always** failing. This code should be refactored to "do what I want" from "do what I mean".

Migrated from https://code.icecube.wisc.edu/ticket/1374
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:07", 
    "description": "http://software.icecube.wisc.edu/static_analysis/00_LATEST/report-c5454a.html#EndPath\n\nThe behavior of `malloc()` when allocating zero bytes is implementation defined. This could result in this section of code **always** failing. This code should be refactored to \"do what I want\" from \"do what I mean\".", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335647931556", 
    "component": "combo simulation", 
    "summary": "muongun - UNIX api violation (malloc)", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-01T13:59:52", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151001T15:04:10: 
* jvansanten commented:

> In changeset 138150:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138150"
> 0-dimensional integration is not a thing. Fixes #1366.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:07: 
* olivas commented:

> Milestone November 2015 deleted


* olivas changed milestone from "November 2015" to ""
